app/uk/gov/hmrc/cdsimportsdds/util/MongoFormatters.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsdds.util
+
+import java.time.Instant
+
+import play.api.libs.json.{JsError, JsNumber, JsObject, JsResult, JsSuccess, JsValue, Json, OFormat}
+import uk.gov.hmrc.cdsimportsdds.models.ImportsDeclaration
+
+object MongoFormatters {
+
+  object ImportsDeclaration {
```

Comment:
why do we need this another nested object to define all the formats?

Reply:
I'll get rid of the outer layer